hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml

```diff
@@ -39,6 +40,11 @@
             <artifactId>spring-data-hazelcast</artifactId>
```

Comment:
I don't think we have a version of `spring-data-hazelcast` ready for 4.0 so upped to 3.12.6.